<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDAFFDialogExport2DPlot</class>
 <widget class="QDialog" name="QDAFFDialogExport2DPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_Browse">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEdit_ExportPath">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox_IncludeAllChannels">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
     </property>
     <property name="text">
      <string>Include all channels</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBox_ShowDataPointDots">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
     </property>
     <property name="text">
      <string>Show dots at data points</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_Export">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
